SerialController/Commands/PythonCommands/ImageProcessingOnly/SV_Auction.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from cgitb import reset
+from Commands.PythonCommandBase import ImageProcPythonCommand
+from Commands.Keys import Button, Hat, Direction, Stick
+import os
+
+class SV_Money(ImageProcPythonCommand):
+
+    NAME = 'SV競り'
+
+    def __init__(self, cam):
+        super().__init__(cam)
+        self.buy_available = False
+
+    def do(self):
+        while True:
+            self.buy_available = False
+            buy_item = os.listdir("./Template/SV/Auction/Item")
+            for i in range(20):
+                if self.isContainTemplate("SV/Auction/play.png", 0.9, show_value=True):
+                    self.buy_available = True
+                    break
+                self.wait(1)
+            if self.buy_available:
+                for i in buy_item:
+                    if self.isContainTemplate(f"SV/Auction/Item/{i}", 0.8, show_value=True):
+                        
+                        self.press(Button.A,0.1,0.5)
+                        while not self.isContainTemplate("SV/Auction/bye.png", 0.8):
+                            self.press(Button.A,0.1,0.8)
+                        self.press(Button.A,0.1,0.5)
+                        self.report()
+                        self.time()
+                        break
+                    if i == buy_item[-1]:
+                        self.resetGame()
+            else:
+                self.report()
+                self.time()
+
+
+
+    def quitGame(self):
+        self.press(Button.HOME, 0.1, 0.5)#HOME
+        self.press(Button.X,0.05,0.5)
+        self.press(Button.A,0.05,0.5)
+        while not self.isContainTemplate("SV/Auction/start.png", 0.8):
+            self.wait(0.5)
+        self.wait(1)
+    
+    def launchGame(self):
+        self.press(Button.A,0.05,1)
+        self.press(Button.A,0.05,1)
+        while not self.isContainTemplate("SV/Auction/title.png", 0.8, show_value=True):
+            self.wait(1)
+            self.press(Button.A, 0.5,1)
```

Comment:
```suggestion
            self.wait(1)
```
あと、ここでAボタン押すとたまにタイトル判定がスキップされて虚無るから、押さない方がよさそう